detox/src/ios/expectTwo.test.js

```diff
@@ -272,6 +272,39 @@ describe('expectTwo', () => {
     expect(testCall).deepEquals(jsonOutput);
   });
 
+  it(`expect(element(by.text('Tap Working!!!'))).toBeFocused()`, () => {
+    const testCall = e.expect(e.element(e.by.text('Tap Working!!!'))).toBeFocused();
+    const jsonOutput = {
+      invocation: {
+        type: 'expectation',
+        predicate: {
+          type: 'text',
+          value: 'Tap Working!!!'
+        },
+        expectation: 'toBeFocused'
+      }
+    };
+
+    expect(testCall).deepEquals(jsonOutput);
+  });
+
+  it(`expect(element(by.text('Tap Working!!!'))).toBeNotFocused()`, () => {
+    const testCall = e.expect(e.element(e.by.text('Tap Working!!!'))).toBeNotFocused();
+    const jsonOutput = {
+      invocation: {
+        type: 'expectation',
+        predicate: {
+          type: 'text',
+          value: 'Tap Working!!!'
+        },
+        modifiers: ['not'],
+        expectation: 'toBeFocused'
+      }
+    };
+
+    expect(testCall).deepEquals(jsonOutput);
+  });
```

Comment:
Not directly related to this feature... But, really, this set of tests surely seems like it should be converted to snapshot tests. @noomorph wdyt?

Reply:
I also agree it would be better